.env.example

```
@@ -36,4 +36,7 @@ SMS_SECRET_KEY=your_sms_secret_key
 SMS_SENDER_NUMBER=01012345678
 
 # 8. Server Configuration
-BASE_URL=http://localhost:8080
\ No newline at end of file
+BASE_URL=http://localhost:8080
+
+#9 OpenAI
+OPENAI_API_KEY=your_openai_api_key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)